### PR TITLE
move items to correct host

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: docs/plans/TODO-2017.md [hunk 1]
```diff
@@ -6,16 +6,15 @@
...
   - [ ] Apache
     - [ ] Finish Shib
       - [ ] Customise login page
       - [ ] Enable Logout [docs](https://wiki.shibboleth.net/confluence/display/IDP30/LogoutConfiguration)
       - [ ] Figure out how we want to manage the metadata
     - [X] Add apache mod
-    - [ ] test sub-domain macro
-    - [ ] enable hsts globally
-    - [ ] merge 000-default w/ 000-ssl
-    - [ ] Migrate sub-Domain
```
Comment (zergless) [inline docs/plans/TODO-2017.md:18]: have these been done? or just not needed?

Comment (butlerx) [reply docs/plans/TODO-2017.md:18]: not needed. since we dont have a suphp replacement we wont be migrating everything to zeus so we dont need them

Comment (butlerx) [reply docs/plans/TODO-2017.md:18]: also hsts will break stuff  in our current setup

**Comment by gruunday** [comment]
This looks okay

**Review by gruunday — APPROVED** [review]

**Pull request merged**